README.md

```diff
@@ -134,86 +222,62 @@ secrets:
   IBEAM_PASSWORD_v1:
     external: true
 
+networks:
+  ib_net_01:
```

Comment:
Yes, it can run on the host network.  I had originally written everything with the intent of running it without a published port, deploying service alongside the IBeam service to handle the trading and using something like haproxy to manage any access required from outside the docker network.

I've revised the documentation to remove the use of a dedicated overlay network.